File: Server/Model/Task.cs
using System;

namespace MyIssue.Server.Model
{
    public class Task
    {
        public decimal TaskId { get; set; }
        public string TaskTitle { get; set; }
        public string TaskDescription { get; set; }
        public string TaskClient { get; set; }
        public string TaskAssignment { get; set; }
        public string TaskOwner { get; set; }
        public string TaskType { get; set; }
        public DateTime? TaskStart { get; set; }
        public DateTime? TaskEnd { get; set; }
        public DateTime TaskCreationDate { get; set; }
        public string CreatedByMail { get; set; }
        public string EmployeeDescription { get; set; }
    }
}

File: Web/ClientApp/src/interfaces/Task.ts
class PagedTaskRequest {
  link: string;
  page: number;
  size: number;
}
class PagedResponse {
  pageNumber: number;
  pageSize: number;
  firstPage: string;
  lastPage: string;
  totalPages: number;
  otalRecords: number;
  nextPage: string;
  previousPage: string;
  data:Task[];
}
class TaskRoot {
  tasks: Task[];
}
class Task {
  taskId: string;
  taskTitle: string;
  taskDescription: string;
  taskClient: string;
  taskAssignment: string;
  taskOwner: string;
  taskType: string;
  taskStart: string;
  taskEnd: string;
  taskCreationDate: string;
  createdByMail: string;
  employeeDescription: string;
}
export {
  PagedResponse,
  PagedTaskRequest,
  TaskRoot,
  Task
  }
